dev_guide/deployments.adoc

```diff
@@ -110,28 +110,61 @@ created each time a new version of the `origin-ruby-sample:latest` image reposit
 ====
 
 == Deployment Client Tool Operations
-Use the 'oc deploy' command with available options to start, view, cancel, or
-retry a deployment.
+
+Use the `oc deploy` command to start, view, cancel, or retry a deployment.
 
 ----
 $ oc deploy <deployment_config> [--options]
 ----
 
-[cols= 2]
-|===
-|`oc deploy <deployment_config> --latest`
-|Manually starts the specified deployment
+=== Start a Deployment [[start-deployment]]
```

Comment:
Can we switch these to gerund verbs, "Starting a Deployment", "Viewing a Deployment", etc?